requirements/customer/goals.tex

```diff
@@ -0,0 +1,27 @@
+\section{Purpose}
+  \subsection{Product Goal}
+    JobAdder aims to facilitate and accelerate the execution of computationally expensive jobs in a cluster of work
+    machines.
+
+    Currently, users need to access every single working machine manually in order to distribute their jobs over them.
+    Even more problems arise when a group of users is involved, in that case, users have to look for the working machine
+    with the least amount of work and add their job to it.
+    
+    JobAdder addresses and solves these problems. With JobAdder, users are able to send their jobs to a central 
+    server in order to evenly distribute them over the working machines. So if there's enough workload (which is usually
+    the case), no machine remains idle unless it's forced to.
+
+    Jobadder also aims to make task scheduling more user-friendly. A user doesn't have to keep checking the state of
+    their job, they'll be notified once the job is done. There is also more insight into the whole system by 
+    providing statistics about user activity, currently running jobs, past jobs and queued jobs.
+
+    Users can also set priorities so that urgent jobs are put before everything, a user is not required to pause or 
+    cancel low-priority jobs manually, the scheduling algorithm takes care of that and starts high-priority jobs in the
+    fastest possible way.
+
+  \subsection{Target Audience}
+    The target audience mostly consists of groups of users with access to several working machines, this includes,
+    companies, academic institutions and computing centers. These users usually need remote machines to run their jobs
+    due to high hardware requirements.
+    
+    JobAdder helps them exploit the full potential of their working machines while keeping things user friendly.
```

Comment:
"while keeping things user friendly" is not precise enough.
How about this:

JobAdder helps them use their hardware more efficiently by adding an abstraction layer between the user and the work machines that are running the user's jobs.